hdfs-audit-parser/hdfs-audit-parser

```diff
@@ -12,8 +12,8 @@
 
 use strict;
 use warnings;
-no warnings 'experimental'; # For smartmatch.
-
+#no warnings 'experimental'; # For smartmatch.
+no if ($] >= 5.018), 'warnings' => 'experimental';
```

Comment:
Your perl skills seem impressive! What does this line do?

Reply:
@arp7  I guess I am just good at breakfix :) I don't claim to have great perl chops.
With perl 5.018, some of the ideas of smartmatch turned out to be a little too smart, and so now perl 5.018 consider it an experimental feature. So simple code, when executed on a 5.018 perl will give warnings. This code change basically takes care to suppress such warnings depending on version.